packages/server/src/modules/websocket/eventHandlers/updatePlayerHandler.ts

```diff
@@ -0,0 +1,50 @@
+import { z } from "zod";
+import { safe } from "../../../lib/fp";
+import { services as playersServices } from "../../players/services";
+import { Socket } from "../types";
+
+export { handleUpdatePlayer };
+
+function handleUpdatePlayer(socket: Socket) {
+  socket.on("updatePlayer", async (args: unknown) => {
+    await handleUpdateHasFinishedStepSafely(socket, args);
+  });
+}
+
+async function handleUpdateHasFinishedStepSafely(
+  socket: Socket,
+  args: unknown
+) {
+  await safe(
```

Comment:
@Baboo7 pk on catch toutes les erreurs?

Reply:
[for the record] cf discussion plus bas dans le fichier [packages/server/src/modules/websocket/eventHandlers/updateGameHandler.ts](https://github.com/dataforgoodfr/offseason_ogre/pull/121/files/24501a38dade9f12b2389fda2cc0ee770ea51b94#diff-cc592307fef0dc2d2c2ef35d90003a2fae1245a94c63ef4fe559373c078298d9)